flexget/components/managed_lists/lists/pending_list/pending_list.py

```diff
@@ -102,12 +107,25 @@ def online(self):
 
     def get(self, entry):
         with Session() as session:
-            match = self._entry_query(session=session, entry=entry, approved=True)
+            match = self._entry_query(session=session, entry=entry, approved=self.only_approved)
```

Comment:
i'm not sure that this will actually include those who are pending, rather than switch select between pending and non pending. If you'd like to select just approved or everything, i'd consider adding the `approved` filter only when needed.

Or, better yet, via enum. Filter by `approved`, `not_approved` or both.